carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml

```diff
@@ -56,7 +56,7 @@ services:
     command: wait-for-it localhost:11311 -- roslaunch /opt/carma/vehicle/config/carma-messenger-docker.launch
 
   messenger_ros2:
-    image: ${DOCKER_ORG}/carma-messenger-core:${DOCKER_TAG}
+    image: usdotfhwastolcandidate/carma-messenger-core:prowler
```

Comment:
So yeah it should be switched where `usdotfhwastolcandidate` hardcode needs to stay as variable. And what we have as variables now needs to be hardcoded as 4.5.0 in my opinion. Lmk if that makes sense

Reply:
updated